data/json/itemgroups/vending_machines.json

```
@@ -130,5 +130,303 @@
     "subtype": "collection",
     "id": "vending_medicine",
     "entries": [ { "group": "drugs_vending", "count-min": 10, "count-max": 20 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "vending_items_adult",
+    "items": [
+      { "item": "condom", "prob": 30, "count": [ 1, 5 ] },
+      { "group": "ss_merch_1", "prob": 20 },
+      { "group": "ss_merch_2", "prob": 20 },
+      { "group": "ss_merch_3", "prob": 20 },
+      { "item": "mag_porn", "prob": 20, "count": [ 1, 3 ] }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "vending_items_useful",
+    "items": [
+      { "item": "umbrella", "prob": 10 },
+      { "item": "teleumbrella", "prob": 10 },
+      { "group": "home_hw", "prob": 50 },
+      { "group": "tools_common", "prob": 50 },
+      { "group": "cleaning", "prob": 30 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "vending_items_pet",
+    "items": [
+      { "group": "petstore_shelves", "prob": 100 },
+      { "group": "petstore_misc", "prob": 100 },
```

`petstore_shelves` contains petpacks and dog armor, so may be best to favor `petstore_misc` which is more narrow.
```suggestion
      { "group": "petstore_misc", "prob": 100 },
```